### PR TITLE
Msa 11548/prasadh

### PR DESCRIPTION
**Pull request by PrasadhNanjundan05**

### CODE REVIEW

File: mano-core-api/src/main/java/com/ubiqube/etsi/mano/service/rest/ManoVnfLcmOpOccs.java [hunk 1]
```diff
@@ -43,10 +44,10 @@ public ManoVnfLcmOpOccs(final ManoClient manoClient) {
 		this(manoClient, null);
 	}
 
-	public List<VnfBlueprint> list() {
+	public List<VnfLcmOpOccs> list() {
```
Comment (Vignaudo) [inline mano-core-api/src/main/java/com/ubiqube/etsi/mano/service/rest/ManoVnfLcmOpOccs.java:47]: I think this class is not in used anymore, `VnfBlueprint` is the correct one.

Comment (PrasadhNanjundan05) [reply mano-core-api/src/main/java/com/ubiqube/etsi/mano/service/rest/ManoVnfLcmOpOccs.java:47]: we dont have any VnfBlueprint class under  etsi-model,  we have only VnfLcmOpOccs

Comment (Vignaudo) [reply mano-core-api/src/main/java/com/ubiqube/etsi/mano/service/rest/ManoVnfLcmOpOccs.java:47]: This one ?

Comment (Vignaudo) [reply mano-core-api/src/main/java/com/ubiqube/etsi/mano/service/rest/ManoVnfLcmOpOccs.java:47]: In mano-core you have to use VnfBluePrint

Comment (PrasadhNanjundan05) [reply mano-core-api/src/main/java/com/ubiqube/etsi/mano/service/rest/ManoVnfLcmOpOccs.java]: Done

**Review by Vignaudo — APPROVED** [review]

**Pull request merged**